docs/source/resources/package-spec.rst

```diff
@@ -378,3 +389,7 @@ numpy-1.8.1-py27_0:
 * numpy >=1.8,<2|1.9
 * numpy 1.8.1 py27_0
 * numpy=1.8.1=py27_0
+
+.. |br| raw:: html
+
+      <br>
```

Comment:
```suggestion

```
This isn't necessary, to my knowledge. Was your local build not looking right at the end?

Reply:
The tables were rendering excessively long on the same line.  So a user would have to scroll across to get all the information.  I didn't know I could allow for breaks from within a table.  It does work so I did remove these.